main.go

```diff
@@ -104,9 +103,6 @@ func main() { //nolint:funlen
 
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
-	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
```

Comment:
May be it could be reasonable to disable it just in the manifests, not in the source code, but no problem to fetch it from the history.

Reply:
i had it thought to add comments , but then feel it is better to make the code neat. 
hopefully one day if anyone wonder why we dont have it there can look for the git history.